common/database/migrations/1681122520981-update_entities_10_04_2023.ts

```diff
@@ -1,20 +1,22 @@
 import { MigrationInterface, QueryRunner } from "typeorm";
 
-export class createFilesEntity1680124728042 implements MigrationInterface {
-    name = 'createFilesEntity1680124728042'
+export class updateEntities100420231681122520981 implements MigrationInterface {
```

Comment:
Please investigate why you have to replace the original migration when you want to create a new migration.